mockcache.py

```diff
@@ -348,23 +353,21 @@ def check_key(key, key_extra_len=0):
         Is an unicode string (Raises MockcachedStringEncodingError)
         Is None (Raises MockcachedKeyNoneError)
     """
-    import types
-    if type(key) == types.TupleType:
+    if type(key) == tuple:
         key = key[1]
     if not key:
-        raise Client.MockcachedKeyNoneError, ("Key is None")
+        raise Client.MockcachedKeyNoneError(("Key is None"))
     if isinstance(key, unicode):
```

Comment:
sure there is unicode (at least in 2.7)

basestring doesn't exists any more in python 3 though that's why I've removed this line